test/v2/controllers/requestParsers/validators/RetrieveBalanceAndTransactionsValidatorSpec.scala

```diff
@@ -46,65 +46,57 @@ class RetrieveBalanceAndTransactionsValidatorSpec extends UnitSpec with MockAppC
       }
 
       "an invalid customer payment information is supplied" in {
-        validator.validate(inputDataDocNumber.copy(customerPaymentInformation = Some("invalid"))) shouldBe List(
-          InvalidCustomerPaymentInformationError)
+        validator.validate(inputDataDocNumber.copy(customerPaymentInformation = Some("invalid"))) shouldBe
+          List(CustomerPaymentInformationFormatError)
       }
 
-      "an invalid include charge estimate is supplied" in {
-        validator.validate(inputDataDocNumber.copy(includeChargeEstimate = Some("invalid"))) shouldBe List(InvalidIncludeChargeEstimateError)
+      "an invalid date from is supplied" in {
+        validator.validate(inputDataDocNumber.copy(fromDate = Some("invalid"))) shouldBe List(V2_FromDateFormatError)
+      }
+      "an invalid date to is supplied" in {
+        validator.validate(inputDataDocNumber.copy(toDate = Some("invalid"))) shouldBe List(V2_ToDateFormatError)
       }
 
-      "multiple invalid values are supplied" in {
-        val input          = inputDataDocNumber.copy(dateFrom = Some("invalid"), removePOA = Some("invalid"))
-        val expectedErrors = List(InvalidDateFromError, InvalidRemovePaymentOnAccountError)
+      "an invalid remove POA is supplied" in {
+        validator.validate(inputDataDocNumber.copy(removePOA = Some("invalid"))) shouldBe List(RemovePaymentOnAccountFormatError)
+      }
 
-        validator.validate(input) shouldBe expectedErrors
+      "an invalid include charge estimate is supplied" in {
+        validator.validate(inputDataDocNumber.copy(includeEstimatedCharges = Some("invalid"))) shouldBe List(IncludeEstimatedChargesFormatError)
       }
 
-      "date from is later than date to" in {
-        val input          = inputDataDateRange.copy(dateFrom = Some("2022-12-01"), dateTo = Some("2022-11-01"))
-        val expectedErrors = List(V2_RangeToDateBeforeFromDateError)
-        validator.validate(input) shouldBe expectedErrors
+      "date to is supplied but date from is not" in {
+        validator.validate(inputDataDateRange.copy(fromDate = None, toDate = Some("2022-11-01"))) shouldBe List(V2_MissingFromDateError)
       }
 
       "date from is supplied but date to is not" in {
-        val input          = inputDataDateRange.copy(dateFrom = Some("2022-12-01"), dateTo = None)
-        val expectedErrors = List(V2_MissingToDateError)
-        validator.validate(input) shouldBe expectedErrors
+        validator.validate(inputDataDateRange.copy(fromDate = Some("2022-12-01"), toDate = None)) shouldBe List(V2_MissingToDateError)
       }
 
-      "date to is supplied but date from is not" in {
-        val input          = inputDataDateRange.copy(dateFrom = None, dateTo = Some("2022-11-01"))
-        val expectedErrors = List(V2_MissingFromDateError)
-        validator.validate(input) shouldBe expectedErrors
+      "date from is later than date to" in {
```

Comment:
Would it make things easier to read if we called this test "fromDate is later than toDate" to match the variable names (and likewise for lines 84 + 88)?